src/components/table/TableView.svelte

```diff
@@ -63,15 +78,34 @@
       {totalPages}
       {currentPage} />
   </div>
+
+  {#if metricsWithPercentileData.includes(data[0].metric_type)}
```

Comment:
(if you implement the above)

```suggestion
  {#if METRICS_WITH_PERCENTILE_DATA.has(data[0].metric_type)}
```